core/src/test/java/com/sequenceiq/cloudbreak/converter/v4/stacks/cluster/CloudStorageConverterTest.java

```diff
@@ -0,0 +1,88 @@
+package com.sequenceiq.cloudbreak.converter.v4.stacks.cluster;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
Please do not use JUnit 4 for new tests; JUnit 5 is preferable.

Reply:
done